packages/ui/src/theme/tokens/components/countryCodeSelect.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
```

Comment:
This doesn't feel like it should be in tokens since there's no countryCodeSelect primitive. Can we move this to the phoneNumberField.js tokens file?

Reply:
Ah, nevermind, I didn't realize that CountryCodeSelect existed.